package.json

```diff
@@ -34,7 +34,7 @@
     "start-brk": "node ./tools/start.js --debug-brk=5858 -enable-logging --v=0 --enable-dcheck --user-data-dir-name=brave-development",
     "test": "cross-env NODE_ENV=test mocha \"test/**/*Test.js\"",
     "testsuite": "node ./tools/test.js",
-    "unittest": "cross-env NODE_ENV=test mocha \"test/unit/**/*Test.js\" --globals chrome,DOMParser,XMLSerializer",
+    "unittest": "python tools/test && cross-env NODE_ENV=test mocha \"test/unit/**/*Test.js\" --globals chrome,DOMParser,XMLSerializer",
```

Comment:
Could we make a new task called `tooltest`? It would be great to isolate `unittest` to purely the JavaScript unit tests 😄

Reply:
or actually, since the output is just stdout (ex: `unittest-cov` wasn't modified), this should be great! Especially since this runs before push 👍